benchmarks/benchmarks/ci/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""Quick running benchmarks to be included in iris-esmf-regrid's CI."""
+
+import numpy as np
+
+from esmf_regrid.esmf_regridder import GridInfo
+
+
+def _make_small_grid_args():
+    """
+    Not importing the one in test_GridInfo - if that changes, these benchmarks
+    would 'invisibly' change too.
+
+    """
+    small_x = 2
+    small_y = 3
+    small_grid_lon = np.array(range(small_x)) / (small_x + 1)
+    small_grid_lat = np.array(range(small_y)) * 2 / (small_y + 1)
+
+    small_grid_lon_bounds = np.array(range(small_x + 1)) / (small_x + 1)
+    small_grid_lat_bounds = np.array(range(small_y + 1)) * 2 / (small_y + 1)
+    return (
+        small_grid_lon,
+        small_grid_lat,
+        small_grid_lon_bounds,
+        small_grid_lat_bounds,
+    )
+
+
+class TimeGridInfo:
```

Comment:
Should this benchmark be contained in the `__init__` file? I was under the impression that for tests, utilities could be contained in the `__init__` but I don't think I've come across actual tests in them.

Reply:
From a functional standpoint there's no problem for ASV, but it seems a good idea to adopt consistent practices, so I've moved things around. See what you think.

cedcaa4438a1786c02bf41a63e58445af369c4b5